File: app/src/main/java/com/bearcave/passageplanning/settings/SettingsFragment.java
package com.bearcave.passageplanning.settings;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

import com.bearcave.passageplanning.R;

/**
 * @author Michał Wąsowicz
 * @version 1.0
 * @since 29.05.17
 */

public class SettingsFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.preferences);
    }

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {

    }

}

File: app/src/main/java/com/bearcave/passageplanning/settings/SettingsFragment.kt
package com.bearcave.passageplanning.settings

import android.os.Bundle
import android.support.v7.preference.PreferenceFragmentCompat

import com.bearcave.passageplanning.R

/**
 * @author Michał Wąsowicz
 * @version 1.0
 * @since 29.05.17
 */

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.preferences)
    }

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {

    }

    companion object {

    }
}
